stratum/hal/lib/barefoot/BUILD

```diff
@@ -66,6 +66,7 @@ stratum_cc_library(
     name = "bf_chassis_manager",
     srcs = ["bf_chassis_manager.cc"],
     hdrs = ["bf_chassis_manager.h"],
+    copts = ["-Wno-thread-safety-analysis"],  # TODO(antonin)
```

Comment:
This is ok, even though gcc does not support it:

> However, if the -Wno- form is used, the behavior is slightly different: no diagnostic is produced for -Wno-unknown-warning unless other diagnostics are being produced.

https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html